core/src/test/scala/integration/kafka/api/AdminClientIntegrationTest.scala

```diff
@@ -354,14 +354,18 @@ class AdminClientIntegrationTest extends IntegrationTestHarness with Logging {
     }
 
     try {
-      TestUtils.waitUntilTrue(() => numMessages.get > 100, "timed out waiting for message produce", 6000L)
+      TestUtils.waitUntilTrue(() => numMessages.get > 10, s"only $numMessages messages are produced before timeout", 6000L)
```

Comment:
Why not increase the timeout instead of decreasing the number of messages? Jenkins' performance is very variable so we should generally be generous with time outs. In fact, I recommend using the default timeout (which we increased until we could get stable builds) unless there's a good reason not to.

Reply:
@ijuma Good point. It is reasonable to simply use the default timeout. I have updated the patch to do that. Thank you.